### PR TITLE
refactor: yurtctl revert use node-servant in dispatch job

### LINKED ISSUE #547: [feature request] yurtctl revert use node-servant in dispatch job
it is a job after https://github.com/openyurtio/openyurt/pull/516

`yurtctl revert` should use `node-servant` now and replace the code.

/help

### PR DESCRIPTION
**Pull request by DrmagicE**
/kind feature

#### What this PR does / why we need it:
This PR makes the following changes:
1. Remove revert cloudnode/edgenode sub command.
2. Replace the "yurt-servant" job in "yurtctl revert" with "node-servant".
3. Revise docs and tutorials.
#### Which issue(s) this PR fixes:

Fixes #547

#### Special notes for your reviewer:

/assign @rambohe-ch @adamzhoul 

#### Does this PR introduce a user-facing change?

```release-note
The subcommand  "cloudnode" and "edgenode" are removed from "yurtctl revert".
Flag "--yurtctl-servant-image" in "yurtctl revert" is replaced by "--node-servant-image".
```

### CODE REVIEW

File: pkg/yurtctl/cmd/convert/convert.go [hunk 1]
```diff
@@ -103,13 +106,16 @@ func NewConvertCmd() *cobra.Command {
 		Short: "Converts the kubernetes cluster to a yurt cluster",
 		Run: func(cmd *cobra.Command, _ []string) {
 			if err := co.Complete(cmd.Flags()); err != nil {
-				klog.Fatalf("fail to complete the convert option: %s", err)
+				klog.Errorf("fail to complete the convert option: %s", err)
```
Comment (adamzhoul) [inline pkg/yurtctl/cmd/convert/convert.go:109]: `klog.Fatalf` will print then calls os.Exit(255).
so why do we update this?

Comment (DrmagicE) [reply pkg/yurtctl/cmd/convert/convert.go:109]: After we upgraded `klog` to v2, `klog.Fatalf` will now print a verbose stack trace, which is not appropriate for a command-line tool, so I change `Fatalf` to `Errorf` and give it a non-zero exit code.

The exit code 1 here is referenced from codes from `kubeadm`:
https://github.com/kubernetes/kubernetes/blob/0153febd9f0098d4b8d0d484927710eaf899ef40/cmd/kubeadm/app/util/error.go#L31

File: pkg/yurtctl/cmd/convert/convert.go [hunk 2]
```diff
@@ -340,6 +348,17 @@ func (co *ConvertOptions) RunConvert() (err error) {
 			edgeNodeNames = append(edgeNodeNames, node.GetName())
 		}
 	}
+	// make sure all cloud nodes and edge nodes are kubernetes nodes
+	for _, v := range edgeNodeNames {
```
[the remark this answers is not in the document]
Comment (DrmagicE) [reply pkg/yurtctl/cmd/convert/convert.go]: @adamzhoul  thanks for pointing out this, yes you are right.  I have removed the snippet.

**Comment by DrmagicE** [comment]
@rambohe-ch Hum..seems the approve command is not working.

**Pull request merged**